libs/ledger-live-common/src/families/cosmos/js-signOperation.ts

```diff
@@ -31,79 +30,47 @@ const signOperation = ({
       let cancelled;
 
       async function main() {
-        const hwApp = new Cosmos(transport);
-
-        const { accountNumber, sequence } = await defaultCosmosAPI.getAccount(
+        const cosmosAPI = new CosmosAPI(account.currency.id);
```

Comment:
Same question as in `prepareTransaction`: could we consider a cache or a map for `CosmosAPI`, to avoid creating new instances every time?